Makefile

```diff
@@ -30,8 +44,13 @@ ansible-galaxy:
 	ansible-galaxy install -r requirements.yml --force
 
 import-gpg-keys:
-	$(foreach var,$(shell cat gpg.recipients | awk -F: '{print $$1}'),gpg --list-public-key $(var) || gpg --keyserver hkp://keyserver.ubuntu.com --search-keys $(var);)
+	$(foreach var, \
+		$(shell cat gpg.recipients | awk -F: '{print $$1}'), \
+		gpg --list-public-key $(var) || gpg --keyserver hkp://keyserver.ubuntu.com --search-keys $(var); \
+	)
 
 recrypt: import-gpg-keys
-	ansible-vault decrypt group_vars/all/secure && pwgen -cynC1 15 | gpg --batch --yes --trust-model always -e -o vault_passphrase.gpg $(shell cat gpg.recipients | awk -F: {'printf "-r "$$1" "'}) && ansible-vault encrypt group_vars/all/secure
-
+	ansible-vault decrypt group_vars/all/secure && pwgen -cynC1 15 | \
+		gpg --batch --yes --trust-model always -e -o vault_passphrase.gpg \
+			$(shell cat gpg.recipients | awk -F: {'printf "-r "$$1" "'}) && \
```

Comment:
Yeah, it was meant to indicate that it's a continuation of the command on the last line, rather than a new command in the pipe/chain. Is it more confusing than helpful?